src/eval/Eval.ml

```diff
@@ -496,11 +512,14 @@ let rec stmt_eval conf stmts =
                 pure @@ ": " ^ pp_literal e_resolved
             | None -> pure ""
           in
-          let err = mk_error1 ("Exception thrown" ^ estr) sloc in
+          let err =
+            mk_error1 ~kind:("Exception thrown" ^ estr) ?inst:None sloc
```

Comment:
Shouldn't `estr` be `inst` here?

Reply:
I missed that, thanks. Sometimes I went for no instance if I wasn't sure.
```suggestion
            mk_error1 ~kind:"Exception thrown" ~inst:estr sloc
```